CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+The PyDy source code is actually spread across both the SymPy and PyDy
+repositories. The two projects are very intimately tied together in terms of
+development. Much of our development practices come from SymPy's methods so
+reading their documentation for contributing is the best place to start:
+
+https://github.com/sympy/sympy/wiki/introduction-to-contributing
+
+We do have some differences though. In particular, we use a checklist to make
+sure each pull request is ready to be merged into the master branch (which
+should always be stable). The checklist is as follows:
```

Comment:
What about "In particular, every pull request into this repository must satisfy a checklist"